src/app/static/src/app/store/stepper/getters.ts

```diff
@@ -18,8 +18,8 @@ export const getters: StepperGetters & GetterTree<StepperState, RootState> = {
     },
     complete: (state: StepperState, getters: any, rootState: RootState, rootGetters: any) => {
         return {
-            1: rootGetters['baseline/complete'] && rootGetters['metadata/complete'],
-            2: rootGetters['surveyAndProgram/complete'],
+            1: rootGetters['baseline/complete'] && rootGetters['metadata/complete'] && rootGetters['surveyAndProgram/complete'],
+            2: rootGetters['baseline/complete'] && rootGetters['metadata/complete'] && rootGetters['surveyAndProgram/complete'],
```

Comment:
Seems a bit odd to include baseline and metadata here, but I guess this is going to get removed in the next ticket anyway.

Reply:
Ignore this comment, this is right enough